packages/react-proptypes/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@stardust-ui/react-proptypes",
```

Comment:
actually, not absolutely convinced that we need to make this split now - could you, please, suggest what are the reasons to introduce this package now? Maybe we could wait for a while with this move - or there is already some client project exists that needs to consume them?